### DISCUSSION TITLE
Wall thickness of 3D design

### DISCUSSION THREAD

**Discussion by eugeniogj** [sn-lab/MouseGoggles#5] [Q&A] [answered]
Hi, 
We are interested in using your headset for our experiments with Two-Photon microscopy. While trying to print the headset with your CAD designs, we were suggested by the 3D printing company to check for the following issues with the file:

_We have detected files [ thin wall thickness (less than 0.8mm) ] in this drawing. Please revise the area of wall thinness(0.15mm) and kindly note that the thin wall thickness area might have the risk of deformation during transportation._

Below are the marked observations in the file.

Would you happen to have any suggestions or an updated file?

![VirtualVR_Aarhus_3DPrint_1](https://user-images.githubusercontent.com/12768210/227913645-63d7cd7d-d943-4fe7-af83-83dfbc01b42c.png)
![VirtualVR_Aarhus_3DPrint_2](https://user-images.githubusercontent.com/12768210/227913649-5a33b5b4-b21f-43f8-b689-cd74a60c8934.png)

**Comment by misaacson01** [comment, marked as answer]
Hello @eugeniogj, thanks for the info. Can I ask what 3D printing service you're using? My first suggestion would be to ask them to print them anyway, as is. Sometimes they have these limits just to make sure people don't get mad if they don't come out perfect or break with too much handling, and are happy to print them anyway if you ask and are aware of the risk. With high-resolution resin-based printers, these parts have printed just fine for us.

If they can't be printed as is, I'll upload some revised versions with thicker walls ASAP, just let me know!

**Comment by eugeniogj** [comment]
Hello @misaacson01, thanks for your quick answer. We are using craftcloud3d. I will let you know how it looks when I receive it!
